MakingItBetterG55/src/application/Player.java

```diff
@@ -0,0 +1,64 @@
+package application;
+
+public class Player {
+	
+	private char playerType;
+	private String playerName;
+	private boolean aiControlled;
+	private int money;
+	private int health;
+	private int education;
+	private int goods;
```

Comment:
maybe add a score variable although this may not be used or at least implemented differently

Reply:
Done.